src/arguments.rs

```diff
@@ -74,7 +76,11 @@ pub enum Command {
         cmd: Option<ConfigSubCommand>,
     },
 }
-
+#[derive(Debug, StructOpt)]
+pub enum EntryType {
```

Comment:
I'd rename this to `Entity`, we use `entities` in general to refer to a bunch of the time-entry related entries.
```suggestion
pub enum Entity {
```

Reply:
done!